test/TransactionURI.spec.ts

```diff
@@ -53,8 +53,8 @@ describe('TransactionURI should', () => {
         const serializedTransaction = 'B600000000000000000000000000000000000000000' +
         '0000000000000000000000000000000000000000000000000000000000000000000000000' +
         '0000000000000000000000000000000000000000000000000000000000000000000000000' +
-        '0000000000000000000000000000190544100000000000000005816E98404000000900FFE' +
-        'A45AEA2EE9B880D5E4F9B91B75857F444F1766CDCB0600010000000000CC403C7A113BDF7' +
+        '0000000000000000000000000000198544100000000000000003BC6450C7E010000983227' +
+        '106BECAC8228ED2A598A8913E207EFB19F31869ECB0600010000000000CC403C7A113BDF7' +
         'C80969800000000000068656C6C6F';
```

Comment:
these changed because previously the epoch adjustment was incorrect?

Reply:
Not particularly - although `Deadline` class has also changed - These changes are because of the Address object recent changes (https://github.com/symbol/symbol-sdk-typescript-javascript/commit/2274bf3d2588b7f29f0fd776d040fe61bcad43bd#diff-8c6fe9f2fcc08e2ae46a11c405b93137998eff3602d29df360eb3f154b761981, the previous address used to produce this payload was not starting with 'T' or 'N' (it was starting with 'M' for `MIJIN_TEST` most probably) therefore it started to fail after the SDK changes. So I just recreated the payload with the correct address to fix it.